__tests__/app.test.js

```diff
@@ -0,0 +1,61 @@
+const request = require("supertest");
+const app = require("../app.js");
+const seed = require("../db/seeds/seed.js");
+const testData = require("../db/data/test-data/");
+const db = require("../db/connection.js");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  db.end();
+});
+
+describe("app", () => {
+  describe("server errors", () => {
+    it("404 GET /api/topics - a path that doesn't exist but is valid format", () => {
+      return request(app)
+        .get("/api/chopics")
+        .expect(404)
+        .then((body) => {
+          const message = body.body.message;
```

Comment:
The reason you're having to use body.body is because you're not destructuring body properly in the brackets of the .then

Instead of getting the property of body, you're naming the whole response body, and it still has a key of body.

So in order to fix this, destructure by putting curly brackets around body